### PR TITLE
Update methodology for choosing preferred VerifiedNumber

### PR DESCRIPTION
**Pull request by gcapalbo**

### CODE REVIEW

File: corehq/apps/sms/management/commands/fix_preferred_number.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+from django.core.management.base import BaseCommand, CommandError
```
Comment (dimagimon) [inline corehq/apps/sms/management/commands/fix_preferred_number.py]: F401 'CommandError' imported but unused

**Comment by gcapalbo** [comment]
@esoergel code review buddy

File: corehq/apps/sms/management/commands/fix_preferred_number.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        # Convert to a list of owner ids that have more than one VerifiedNumber
+        # (excluding pending numbers)
+        mult_list = []
+        for owner_id, count in owners.iteritems():
+            if count > 1:
+                owner_vns = VerifiedNumber.view(
+                    'sms/verified_number_by_owner_id',
+                    key=owner_id,
+                    include_docs=True
+                ).all()
```
Comment (esoergel) [inline corehq/apps/sms/management/commands/fix_preferred_number.py]: Do you expect there to be a lot of these?

Comment (gcapalbo) [inline corehq/apps/sms/management/commands/fix_preferred_number.py]: nope, there's just like 2-3 numbers per user, and there's < 50 users we need to run this for

File: corehq/apps/sms/management/commands/fix_preferred_number.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        ).all()
+
+        # Convert to a dict of {owner id: count of total numbers}
+        owners = {}
+        for vn in vns:
+            owner_id = vn['key']
+            if owner_id in owners:
+                owners[owner_id] += 1
+            else:
+                owners[owner_id] = 1
```
Comment (esoergel) [inline corehq/apps/sms/management/commands/fix_preferred_number.py]: Shame there's no `count.js` for this view

**Pull request merged**